config/manager/kustomization.yaml

```diff
@@ -12,5 +12,4 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/openshift_sandboxed_containers/openshift-sandboxed-containers-operator
-  newTag: 1.3.1
+  newName: quay.io/bpradipt/openshift-sandboxed-containers-operator
```

Comment:
This looks like a local modification, I guess it shouldn't be part of this PR?

Reply:
Right.